modules/microservice/main.tf

```diff
@@ -453,7 +453,7 @@ resource "azurerm_app_service" "microservice" {
       active_directory  {
           client_id = azuread_application.microservice.application_id
           allowed_audiences = [ 
-            "https://${var.name}-${each.value.location}-${var.environment_name}${local.appservices_baseurl}",
+            "https://${var.service_name}-${var.name}-${each.value.location}-${var.environment_name}${local.appservices_baseurl}",
```

Comment:
Same recommendation as in the other PR to make this a configurable option at the root so existing implementation will not be broken. 

I like having this new option for the conventions :)

Reply:
I like that option.  Maybe add a top level option / variable of 'appservice_naming_convention_type' with two valid options of short and long?  Additionally, add a similar option for keyvault_naming_convention_type ?

```
variable "appservice_naming_convention_type" {
  description = "Type of App Service naming convention to follow"
  type        = string
  default     = "short"
  validation {
    condition     = contains(["short", "long"], lower(var.azure_environment))
    error_message = "Not a valid naming convention."
  }
}
```

Reply:
I was thinking along the lines of a bool that would include service or not, but this is better and opens up for other conventions :)